.github/actions/install-dashboards/action.yml

```diff
@@ -15,7 +15,6 @@ outputs:
     description: "The directory where the plugin has been configured"
     value: ${{ steps.determine-plugin-directory.outputs.plugin-directory }}
 
-
```

Comment:
@RyanL1997 Can you please add the empty space back?

Reply:
Hi @aoguan1990. Thx for the review! Sure, I will fix that in the next commit~